### PR TITLE
Remove Bob as author of ECIP-1072

### PR DESCRIPTION
**Pull request by bobsummerwill**
He did not author this ECIP.
There was no consent to add him.
He actively opposes this ECIP, and supports ECIP-1061.

### CODE REVIEW

File: _specs/ecip-1072.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ title: Aztlán EVM and Protocol Upgrades (Yingchun Edition)
 status: Last Call
 review-period-end: 2019-12-21
 type: Meta
-author: Wei Tang (@sorpaas), Talha Cross (@soc1c), Bob Summerwill (@bobsummerwill)
+author: Wei Tang (@sorpaas), Talha Cross (@soc1c)
```
Comment (meowsbits) [inline _specs/ecip-1072.md]:
```suggestion
author: Wei Tang (@sorpaas)
```

**Comment by bobsummerwill** [comment]
There is no reason for this to be closed.

I am listed as an author and so is Afri and our consent was not given.

**Comment by sorpaas** [comment]
Superseded by https://github.com/ethereumclassic/ECIPs/pull/212

**Comment by bobsummerwill** [comment]
Gotcha.  Thanks for the clarification.

**Pull request closed without merging**